_data/hosts.yml

```diff
@@ -548,6 +548,22 @@
             semver: 7.1.1    
     last_scanned_at: '2017-01-09T15:57:19+0000'
 -
+    name: 'GalacSYS'
+    url: 'https://www.galacsys.net/hebergement_mutualise_details.html'
+    type: shared
+    default: null
+    versions:
+        44:
```

Comment:
It's not a typo, that's really PHP4.4 (see http://php4info.galacsys.com/).
I assume this is intended for legacy websites made by Mathusalem himself 👴🏼 ?